web/html/src/utils/test-utils/forms.ts

```diff
@@ -14,3 +19,45 @@ export const getFieldValuesByName = (formName: string, fieldName: string) => {
   const fields = queryAllByName(form, fieldName);
   return fields.filter(isInput).map(field => field.value);
 };
+
+const asyncAnimationFrame = () => new Promise(resolve => window.requestAnimationFrame(() => resolve(undefined)));
+
+/**
+ * This is a usable alternative for @testing-library/user-event's `type()`.
+ * The library;s `type()` is non-deterministic without a delay and prohibitively
```

Comment:
A very dumb comment, and it is due to a copy-paste. Typo `library;s` instead of `library's`

Reply:
Thanks, I originally wrote the comment anyway, so it's all water in my bucket. :D